e2e-pw/src/oss/specs/sidebar/sidebar-cifar.spec.ts

```diff
@@ -29,9 +29,16 @@ test.describe("classification-sidebar-filter-visibility", () => {
   test("In cifar grid, setting visibility directly works", async ({
     grid,
     sidebar,
+    eventUtils,
   }) => {
     // Test the visibility mode:
     await sidebar.toggleSidebarMode();
+    // mount eventListener
+    const gridRefreshedEventPromise =
+      eventUtils.getEventReceivedPromiseForPredicate(
+        "re-render-tag",
+        (e) => e.detail === "re-render-tag"
```

Comment:
Same here. You need not pass anything for event detail since it's redundant, in which case the predicate would be `() => true`